cogs/demoHelper.py

```diff
@@ -0,0 +1,130 @@
+from discord.ext import commands
+from discord.ext.commands import Context
+
+import logging
+import shelve
+
+from helpers import listPrint
+
+queues = {'dummy': []}
+adminRoles = ['Demonstrator','demonstrator','Admin role','ADMIN ROLE','DEMONSTRATOR','admin role','adminrole']
+
+
+def setup(bot):
+    """
+    Setup the cogs in this extension
+    """
+    bot.add_cog(Students(bot))
+    bot.add_cog(Demonstrators(bot))
+
+
+def getQueue(serverName: str):
+    """
+    Get the relevant queue for the server
+    :param serverName: the server you want the queue for
+    :return: The queue for the server
+    """
+    if serverName in queues.keys():
+        return queues.get(serverName)
+    else:
+        queues[serverName] = []
+        return queues.get(serverName)
+
+
+def getCustomAddMessage(serverName: str):
+    """
+    Gets the custom add message for the server
+    :param serverName: the server you want the queue for
+    :return: The add message for this server.
+    """
+    with shelve.open('addMessage.shelve') as db:
+        if str(serverName) in db:
+            return db.get(str(serverName))
+        else:
+            return ''
+
+
+class Demonstrators(commands.Cog):
+    """
+    Commands for demonstrators and Admins.
+    """
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    @commands.has_any_role(*adminRoles)
+    async def setAddMessage(self, ctx: Context, *, message: str):
+        """
+        Change the default add message.
+        :param message: The new add message to set.
+        """
+        logging.info('{0} setAddMessage {1}'.format(ctx.guild, message))
+        with shelve.open('addMessage.shelve') as db:
+            db[str(ctx.guild)] = message
+        await ctx.send('Anyone added to queue will see this msg:\n' + message)
+
+    @commands.command()
+    @commands.has_any_role(*adminRoles)
+    async def next(self, ctx: Context):
+        """
+        Get the next student in the queue.
+        """
+        if len(getQueue(ctx.guild)) > 0:
+            next = getQueue(ctx.guild).pop(0)
+            logging.info('{0} next {1}'.format(ctx.guild, next))
+            if next is not None:
+                await ctx.send(
+                    'The next student in the queue is {0}, {1} will be with you shortly to signoff or help you.'.format(
+                        next, ctx.message.author.mention))
+        else:
+            await ctx.send('No more students in the queue.')
+
+    @commands.command()
+    @commands.has_any_role(*adminRoles)
+    async def print(self, ctx: Context):
+        """
+        Print out the students in the queue.
+        """
+        logging.info('{0} queue {1}'.format(ctx.guild, getQueue(ctx.guild)))
+        queue = getQueue(ctx.guild)
+        if queue is None or len(queue) == 0:
+            await ctx.send('No students in the Queue!')
```

Comment:
"No students in the Queue" would be better here I think as it sounds more formal without the "!" but feel free to leave it in